cmd/tsbs_generate_queries/databases/cassandra/devops.go

```diff
@@ -50,7 +50,10 @@ func (d *Devops) getHostWhere(nHosts int) []string {
 // AND time >= '$HOUR_START' AND time < '$HOUR_END'
 // GROUP BY minute ORDER BY minute ASC
 func (d *Devops) GroupByTime(qi query.Query, nHosts, numMetrics int, timeRange time.Duration) {
-	interval := d.Interval.RandWindow(timeRange)
+	interval, err := d.Interval.RandWindow(timeRange)
+	if err != nil {
+		panic(err.Error())
```

Comment:
We could wrap these errors for additional readability. Especially since we are panicking on them.

Reply:
Happy to do that, thought this was a simpler change with less chance of me getting it wrong.

Reply:
Its not a big deal, we can leave it for later when they introduce error wrapping to `stdlib`.